### DISCUSSION TITLE
How to add a file icon for an avalonia application?

### DISCUSSION THREAD

**Discussion by sectlying** [AvaloniaUI/Avalonia#9743] [Q&A] [answered]
![QQ截图20221220083719](https://user-images.githubusercontent.com/77864650/208554617-8e205530-5192-40cb-b2ec-eac8afcd274d.png)
I want to custom it but I can't find manual.

**Comment by timunie** [comment, marked as answer]
this is not Avalonia specific. You can set it in your project settings or porject file, see: https://stackoverflow.com/questions/59309154/how-add-icon-to-dotnet-desktop-windows-application for reference.
